.github/workflows/update_uplink_schema.yml

```diff
@@ -0,0 +1,32 @@
+name: Update Uplink Schema
+run-name: ${{ github.actor }} is updating the Uplink schema 🚀
+on:
+  schedule:
+    # On the 5 of every hour
+    - cron:  '5 * * * *'
+jobs:
+  Update-Uplink-Schema:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rover
+        run: |
+          curl -sSL https://rover.apollo.dev/nix/v0.14.1 | sh
+          echo "$HOME/.rover/bin" >> $GITHUB_PATH
+      - name: Make changes to pull request
+        run: |
+          rover graph introspect https://uplink.api.apollographql.com/ | perl -pe 'chomp if eof' > ./apollo-router/src/uplink/uplink.graphql
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v5
```

Comment:
I was wondering, but the README of https://github.com/peter-evans/create-pull-request says it has logic for not opening a PR if nothing has changed or if there’s already/still one open for that branch name.

Reply:
Correct, and I've verified that works on https://github.com/abernix/router/pull/2.